server/controllers/index.js

```diff
@@ -1,5 +1,8 @@
 module.exports = function initControllers(libs, conf, managers) {
   const controllers = {};
 
+  require('./restore')(managers, controllers);
+  require('./save')(managers, controllers, libs);
```

Comment:
@Grenadingue
As it will use different views and it doesn't do the same thing, I think it is better to separate it.
Furthermore, both controllers for save et restore are pretty big.